webviews/editorWebview/index.css

```diff
@@ -426,58 +445,66 @@ body .overview-title .button-group button:last-child {
 }
 
 .section {
-	padding-bottom: 20px;
+	padding-bottom: 24px;
+	border-bottom: 1px solid var(--vscode-editorWidget-border);
+	display: flex;
+	flex-direction: column;
+	gap: 12px;
+}
+
+.section:last-of-type {
+	padding-bottom: 0px;
+	border-bottom: none;
```

Comment:
This leads to a border bellow the labels section but not below the milestone section if these sections are displayed above the comments instead of in a sidebar at the right. IMO there should be no border bellow the labels section in this case.

![image](https://user-images.githubusercontent.com/46387399/195886283-2bdd0355-a191-4551-8b81-ac908ae0eecc.png)